src/__tests__/app.test.ts

```diff
@@ -20,32 +23,41 @@ describe("app", () => {
 
   describe("GET /address-book-io/v1/digital-address/courtesy", () => {
     it("should return a 400 error if the x-pagopa-cx-taxid header is not provided", async () => {
-      const res = await app.get(endpoint);
+      const res = await app.get(endpoint).set({ "x-api-key": "key" });
       expect(res.status).toBe(400);
     })
 
     it("should return a 200 if the x-pagopa-cx-taxid header is provided", async () => {
-      const res = await app.get(endpoint).set({ "x-pagopa-cx-taxid": "example" });
-      expect(UserActivationDocument).toHaveBeenCalledTimes(1);
+      const res = await app.get(endpoint).set({ "x-pagopa-cx-taxid": aFiscalCode, "x-api-key": "key" });
       expect(res.status).toBe(200);
     })
+
+    it("should return a 403 error if the x-api-key header is not provided", async () => {
+      const res = await app.get(endpoint);
+      expect(res.status).toBe(403);
```

Comment:
Can we add this?
```suggestion
      expect(UserActivationDocument).not.toBeCalled();
      expect(res.status).toBe(403);
```
Here and in other failure tests where the User activation should not be called.

Reply:
done in https://github.com/pagopa/io-pn-mock/pull/2/commits/4eb86e5f0fa2c2d13fe42cc57a20254ca4631e84